idunn/places/pj_poi.py

```diff
@@ -210,3 +215,185 @@ def get_raw_grades(self):
 
     def get_reviews_url(self):
         return self.get("Links", {}).get("viewReviews", "")
+
+
+class PjApiPOI(BasePlace):
+    PLACE_TYPE = "poi"
+
+    def __init__(self, d: Union[pj_find.Listing, pj_info.Response]):
+        super().__init__(d)
+        self.data = d
+
+    def get_id(self):
+        listing_id = self.data.listing_id
+        return f"pj:{listing_id}" if listing_id else None
```

Comment:
The small id used by Qwant Maps is actually the `merchant_id` (composed of 8 digits).